docs/modules/admin_initial_setup/pages/carousels.adoc

```diff
@@ -20,11 +20,11 @@ The Carousel Types administrative interface is used to create, edit, or delete c
 
 There are five stock Carousel Types:
 
-* *Newly Cataloged Items* - titles appear automatically based on the active date of the title’s copies
-* *Recently Returned Items* - titles appear automatically based on the mostly recently circulated copy’s check-in scan date and time 
-* *Top Circulated Titles* - titles appear automatically based on the most circulated copies in the Item Libraries identified in the carousel definition; titles are chosen based on the number of action.circulation rows created during an interval specified in the carousel definition and includes both circulations and renewals
-* *Newest Items by Shelving Location* - titles appear automatically based on the active date and shelving location of the title’s copies 
-* *Manual* - titles are added and managed manually by library staff
+* *Newly Cataloged Items* - titles appear automatically based on the active date and shelving location of the title’s copies.  Shelving locations are required to create this carousel.
+* *Recently Returned Items* - titles appear automatically based on the mostly recently circulated copy’s check-in scan date and time.  Shelving locations are not required to create this carousel.
```

Comment:
I think shelving locations might be required for the "Recently returned" one to automatically update, based on the [config in the database](https://github.com/evergreen-library-system/Evergreen/blob/3663817ec8c975d09150fef4c578e1fbd7ae0312/Open-ILS/src/sql/Pg/upgrade/1175.schema.carousels.sql#L19) and the [perl script's handling of that config](https://github.com/evergreen-library-system/Evergreen/blob/518f9a2f7a383c0cff780bdf9fc33e7e90ffe5bc/Open-ILS/src/perlmods/lib/OpenILS/Application/Storage/Publisher/container.pm#L148-L151)